SnackOverflow/src/main/java/ro/utcn/sd/vasi/SnackOverflow/controller/ConsoleController.java

```diff
@@ -0,0 +1,96 @@
+package ro.utcn.sd.vasi.SnackOverflow.controller;
+
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import ro.utcn.sd.vasi.SnackOverflow.model.Answer;
+import ro.utcn.sd.vasi.SnackOverflow.model.Question;
+import ro.utcn.sd.vasi.SnackOverflow.model.User;
+import ro.utcn.sd.vasi.SnackOverflow.model.UserData;
+import ro.utcn.sd.vasi.SnackOverflow.services.AnswerManagementService;
+import ro.utcn.sd.vasi.SnackOverflow.services.QuestionManagementService;
+import ro.utcn.sd.vasi.SnackOverflow.services.UserManagementService;
+
+import java.util.*;
+
+@Component
+@Data
+@RequiredArgsConstructor
+@Profile("!test")
+public class ConsoleController implements CommandLineRunner {
+    private final Scanner scanner = new Scanner(System.in);
+    private final AnswerManagementService answerManagementService;
+    private final QuestionManagementService questionManagementService;
+    private final UserManagementService userManagementService;
+    private final List<CommandHandler> commandHandlers = new ArrayList<>();
```

Comment:
You can actually transform the handlers into `@Components` and inject all of them (via Spring). See https://vladmihalcea.com/why-i-like-spring-autowired-for-list-types/